test/bundle.c

```diff
@@ -163,23 +163,20 @@ static void bundle_test_resign(BundleFixture *fixture,
 	resignbundle = g_build_filename(fixture->tmpdir, "resigned-bundle.raucb", NULL);
 	g_assert_nonnull(resignbundle);
 
-	/* Switch to release key pair */
-	r_context_conf()->certpath = g_strdup("test/openssl-ca/rel/release-1.cert.pem");
-	r_context_conf()->keypath = g_strdup("test/openssl-ca/rel/private/release-1.pem");
+	/* Input bundle must *not* verify against 'rel' keyring */
+	r_context()->config->keyring_path = g_strdup("test/openssl-ca/rel-ca.pem");
```

Comment:
Please explain why we need to use r_context instead of r_context_conf in this case.